Lib/openbakery/profiles/name.py

```diff
@@ -419,12 +419,11 @@ def com_google_fonts_check_name_match_familyname_fullfont(ttFont):
 
 
 @check(
-    id="com.google.fonts/check/family_naming_recommendations",
-    proposal="legacy:check/071",
+    id="com.adobe.fonts/check/postscript_name",
+    proposal="https://github.com/miguelsousa/openbakery/issues/62",
 )
-def com_google_fonts_check_family_naming_recommendations(ttFont):
-    """Font follows the family naming recommendations?"""
-    # See http://forum.fontlab.com/index.php?topic=313.0
+def com_adobe_fonts_check_postscript_name(ttFont):
+    """PostScript name follows Adobe requirements?"""
```

Comment:
This check is in the **name.py** file, which is part of the `OpenType` profile (which is included by default on the `Universal` profile). So it should not be described as "Adobe" requirements.

If the check is generally useful (not vendor specific) then it should be described merely as "typical requirements" (or actually, in this case, "Open Type spec requirements")